.github/workflows/ci.yml

```diff
@@ -18,10 +18,29 @@ jobs:
         with:
           fetch-depth: 2
 
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18'
+
       - name: Install pnpm
+        uses: pnpm/action-setup@v2
+        with:
+          version: 'latest'
```

Comment:
We need to explicitly put a specific version here that matches the `package.json`'s `packageManager` field. Currently specified at `10.5.2`, once the `latest` tag updates to a new version, the `CI` will fail.